vm/src/builtins/object.rs

```diff
@@ -14,9 +14,6 @@ use crate::{
     PyContext, PyObject, PyObjectRef, PyResult, PyValue, TryFromObject, TypeProtocol,
 };
 
-/// object()
-/// --
-///
```

Comment:
As I understood, `ImplItem<Item> for MethodItem` will not affect at class documentation `ClassItem`. When I executed `object.__text_signature__`, it returns `None`. I guess these changes will revert some resolved problems again (https://github.com/RustPython/RustPython/issues/2732#issuecomment-898936219, https://github.com/RustPython/RustPython/issues/2623#issuecomment-898862717).